mono/metadata/native-library.c

```diff
@@ -696,6 +697,7 @@ pinvoke_probe_for_module (MonoImage *image, const char *new_scope, char **found_
 			break;
 	}
 
+leave:
```

Comment:
nit: I prefer `exit` in new code. It is shorter, and does not interfere with the sometimes `#define leave __leave` you see on Windows.